cel/src/lib.rs

```diff
@@ -233,6 +233,7 @@ mod tests {
 
         // Test methods
         assert_output("size([1, 2, 3]) == 3", Ok(true.into()));
+        assert_output("size([size([42]), 2, 3]) == 3", Ok(true.into()));
```

Comment:
I _think_ this is an example of where you'd need to look the function up previously.